tox.ini

```diff
@@ -1,29 +1,43 @@
 [tox]
-envlist = py{3.6,3.7,3.8}-flask{1}
+envlist = py{3.6,3.7,3.8,3.9}-flask{1,2}
 isolated_build = True
 
 [testenv]
-whitelist_externals = poetry
 setenv =
     FLASK_APP = tests/integration/app.py
+skip_install = true
 deps =
     flask1: flask>=1.0.0,<2.0.0
-commands =
-    poetry install
-    poetry run black . --check
-    poetry run flake8 flask_alchemydumps/ tests/
-    poetry run nosetests
+    flask2: flask>=2.0.0,<3.0.0
+    flask-sqlalchemy==2.4.1
+    python-decouple==3.3
+    sqlalchemy==1.3.16
+    black==19.10b0
+    flake8==3.7.9
```

Comment:
If you ate creating a step (an env) only, why do we install linting tools everywhere?